aten/src/TH/generic/THStorageCopy.cpp

```diff
@@ -8,7 +8,7 @@ void THStorage_(copy)(THStorage *storage, THStorage *src)
   scalar_t *scalar_src = THStorage_(data)(src);
   scalar_t *data = THStorage_(data)(storage);
   uint64_t numel = storage->nbytes() / sizeof(scalar_t);
-  for (ptrdiff_t i = 0; i < numel; ++i) {
+  for (uint64_t i = 0; i < numel; ++i) {
```

Comment:
nit: can `ptrdiff_t` size change based on platform?

Reply:
Yes, it is 32-bit on 32-bit OSes, but I don't think PyTorch supports any at the moment.